utils/.config.template.toml

```diff
@@ -33,9 +33,10 @@ resolution_w = { optional = false, default = 1080, example = 1440, explantation
 resolution_h = { optional = false, default = 1920, example = 2560, explantation = "Sets the height in pixels of the final video" }
 
 [settings.background]
-background_choice = { optional = true, default = "minecraft", example = "rocket-league", options = ["minecraft", "gta", "rocket-league", "motor-gta", "csgo-surf", "cluster-truck", "minecraft-2","multiversus","fall-guys","steep", ""], explanation = "Sets the background for the video based on game name" }
-#background_audio = { optional = true, type = "bool", default = false, example = false, options = [true, false,], explanation = "Sets a audio to play in the background (put a background.mp3 file in the assets/backgrounds directory for it to be used.)" }
-#background_audio_volume = { optional = true, type = "float", default = 0.3, example = 0.1, explanation="Sets the volume of the background audio. only used if the background_audio is also set to true" }
+background_video = { optional = true, default = "minecraft", example = "rocket-league", options = ["minecraft", "gta", "rocket-league", "motor-gta", "csgo-surf", "cluster-truck", "minecraft-2","multiversus","fall-guys","steep", ""], explanation = "Sets the background for the video based on game name" }
+background_audio = { optional = true, default = "lofi", example = "minecraft", options = ["minecraft","lofi","undertale"], explanation = "Sets a audio to play in the background (put a background.mp3 file in the assets/backgrounds directory for it to be used.)" }
+background_audio_volume = { optional = true, type = "float", nmin = 0, nmax = 1, default = 0.1, example = 0.05, explanation="Sets the volume of the background audio. Type a value between 0 and 1.", oob_error = "The volume HAS to be between 0 and 1", input_error = "The volume HAS to be a float number between 0 and 1"}
+allow_only_tts = { optional = true, type = "bool", default = false, example = false, explanation="Used if you want to render another video with only TTS audio in a separate folder", input_error = "The value HAS to be true or false"}
```

Comment:
`allow background audio` should be a setting not `allow_only_tts`

Reply:
`allow_only_tts` isn't a config that allow the background audio. If the user don't want the background audio, just set `background_audio_volume` to 0. `allow_only_tts` allow the creation of OnlyTTS folder that will render a new video without the background_audio.

If you want a video without a background_audio: Set 'background_audio_volume' to 0.
If you want a video with background_audio: Set 'background_audio_volume' to something like 0.1 and set 'allow_only_tts' to 'false'
If you want 2 above: Set volume to something like 0.1 and Set 'allow_only_tts' to 'true'

Reply:
In the screenshots you can see that some files was generated with and without the background_audio. A copy without background audio was renderized to an separated folder

Reply:
it can be more simple 
for example:
>If you want a video without a background_audio: Set 'background_audio_volume' to 0.
If you want a video with background_audio: Set 'background_audio_volume' to something like 0.1 and set 'allow_only_tts' to 'false'
If you want 2 above: Set volume to something like 0.1 and Set 'allow_only_tts' to 'true'
 
this is confusing me so how user gonna digest what is going on .

Reply:
it could be `allow background_audio` as bool and back_audio_multiplier or something as float

Reply:
Hey Electro, did you try to run my fork? I think it's simple because my insight allows the user to render two videos - one with background audio and one without it if the user wants. This is useful because on platforms like TikTok, you can add the background music of the trend. If the user wants, they can render two types of videos - one for posting on TikTok and another for posting on YouTube, for example.

Reply:
If you want to give this option a simple name, that's fine. However, if the user doesn't want the background audio, they can simply set the background_audio_volume to 0. In the chop_background function, the program will skip the audio creation.

Reply:
By default, `allow_only_tts` is set to `false`, so the average user will skip this optional parameter in the TOML creation.

Reply:
I was prompting with chatgpt and i think the best recomendation for this setting is `enable_extra_audio`.
By default it will be false, so don't worry about new users. If they didn't know how to use. Just don't use.
If they don't want backgrond audio, just set the volume to 0 that will be skipped background audio creation.